### ISSUE TITLE
Minor things

### ISSUE THREAD

**Issue by grantmcdermott** [stata2r/stata2r.github.io#1]
Site looks great so far. One or two minor things worth deciding on before we start adding more content..

- **Assignment.** This is a constant refrain for me, but I'm a big believer that R tutorials (esp. ones aimed at users coming from other languages) should use `=` instead of `<-`. What's more, I know that both the data.table team and Laurent subscribe to this notation in all of their respective vignettes and documentation. So I think we should use `=` for both simplicity and consistency with the underlying docs.

- **Consistent data name(s)**. Should we use a consistent shorthand for the hypoethical dataset across the two pages? At the moment it's (most) `data` for the fixest page and `df` for the data.table page. I'm okay with that principle, but maybe worth discussing. (TBH I'm not the _hugest_ fan of either `data` and `df` b/c they both overload function names and thereby invite the infamous *object-of-type-closure-is-not-subsettable* error.... but maybe no biggie for a simple tutorial.)

**Comment by NickCH-K** [comment]
While I generally prefer `<-` I'm fine to go with `=` on this.

Agreed that the data set should have a consistent name, and that `data` and `df` both have their own problems. How about `d`?

Along those lines it might be nice to have consistent _variable_ names too. Sticking with what's already on the regression page, maybe `x` etc. for numeric variables, `cat` for factors, `unit` for ID variables and `time` for time, in all cases expanding to `x1`, `x2` or `unit1`, `unit2` etc. if we need multiples? Character variables will probably pop up on the data cleaning page, maybe `str` for those?

**Comment by kylebutts** [comment]
I agree with `d` for data. That way the code will be concise

As of last night, every example in data cleaning I had was using the same dataset. I like the idea of being able to copy and paste each example and having it run.

That is not the case for the regression section, but I was thinking of downloading an ACS sample or something like that so everything can run.

**Comment by grantmcdermott** [comment]
Cool, I'm good with `d` (or maybe `dat`?) as long as we add an explainer at the top. On that note, we should also mention some of the macro features at the top of the page. E.g. `setFixest_estimation(data = d)` so that you don't have to reference your dataset each time. I think Stata users will like that one in particular.

PS -- Similarly for `setFixest_vcov`. At the moment, our opening example will give different SEs b/c fixest will default to clustered SEs. We should probably start off with no fixed effects, then introduce SE adjustment, and then get on to FEs...

**Comment by NickCH-K** [comment]
More minor things:

- There are big gaps on the page between the different sections. Could those come down?
- I've added a to-do list in comments at the bottom of the `data.table` page, for stuff to add to that page, might be handy to have one for `fixest` too
- I can't get the index page to build, even in its current form. Is there some encoding thing I need to do in RStudio to handle the emojis? Alternately if one of you could add this paragraph and build the index that work too:

```r
p("Plus, they are ", span(class = "italic", "flexible"), " (👌✨) and can help you avoid R's 'package overload.' Depending on what you do in Stata, you might even be able to cut back, going from three packages - ", span(class = "font-semibold", "reghdfe, estout"), " and ", span(class = "font-semibold", "gtools"), " to just two."))
```

**Comment by grantmcdermott** [comment]
Most of these have been taken care of. Closing in favour of https://github.com/stata2r/stata2r.github.io/issues/7
